test/parallel/test-tls-disable-renegotiation.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 const common = require('../common');
 const assert = require('assert');
-const fs = require('fs');
+const fixtures = require('../common/fixtures');
```

Comment:
Nit: can you please move this and the `assert` require after the `common.hasCrypto` check?